src/modint.rs

```diff
@@ -898,6 +898,39 @@ macro_rules! impl_basic_traits {
             }
         }
 
+        #[cfg(feature = "num-traits")]
+        impl<$generic_param: $generic_param_bound> num_traits::Zero for $self {
+            #[inline]
+            fn zero() -> Self {
+                Self::new(0)
+            }
+            #[inline]
+            fn is_zero(&self) -> bool {
+                self == &Self::zero()
+            }
+        }
+
+        #[cfg(feature = "num-traits")]
+        impl<$generic_param: $generic_param_bound> num_traits::One for $self {
+            #[inline]
+            fn one() -> Self {
+                Self::new(1)
+            }
+            #[inline]
+            fn is_one(&self) -> bool {
+                self == &Self::one()
+            }
```

Comment:
```suggestion
```

This part can be omitted (unlike `Zero`, somehow).

Reply:
Huh, that's quite surprising :)  However, I took a look at [the doc](https://docs.rs/num-traits/latest/num_traits/identities/trait.One.html#method.is_one) to realize that it actually recommends to implement this.  So shall we just leave this as it is?